.github/workflows/release.yml

```diff
@@ -20,7 +25,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        nif: ["2.17", "2.16", "2.15"]
```

Comment:
We don't need to build for `2.17` yet, because we don't use anything specifically from that version. Rustler Precompiled will select the `2.16`.